portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,18 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+// Automatic Slideshow - change image every 3 seconds
+var myIndex = 0;
+carousel();
+
+function carousel() {
+  var i;
```

Comment:
use let instead of var, var has similar problems to global variables https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
fixed - using let instead of var inside the function

Reply:
I actually fixed these details earlier today, but got to reply on comments just now :)